projects/Unit/APIApproval.Approve.verified.txt

```diff
@@ -638,9 +637,9 @@ namespace RabbitMQ.Client.Events
     }
     public class CallbackExceptionEventArgs : RabbitMQ.Client.Events.BaseExceptionEventArgs
     {
-        public CallbackExceptionEventArgs(System.Exception e) { }
```

Comment:
Public API change, but I don't think this is in the area where someone would actually care much. (Should be fine for 7.0)

Reply:
Currently, the delta with 7.0 is large enough that this likely won't merge cleanly.

Reply:
=> https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/986#issuecomment-756449610

Reply:
I agree that this is not the kind of public API change that almost anyone would ever notice.